Reqnroll.Generator/TestGenerator.cs

```diff
@@ -139,8 +142,9 @@ private string FixVBNetAsyncMethodDeclarations(string generatedTestCode)
             return result;
         }
         
-        private CodeNamespace GenerateTestFileCode(FeatureFileInput featureFileInput)
+        private CodeNamespace GenerateTestFileCode(FeatureFileInput featureFileInput, out IEnumerable<string> generationWarnings)
         {
+            generationWarnings = new List<string>();
```

Comment:
Could we only assign this when we return?
Could we use Array.Empty?

Reply:
I've changed this to:
```c#
            generationWarnings = null;
```